pages/contact.js

```diff
@@ -11,9 +12,23 @@ export default class Contact extends Component {
         <Helmet
```

Comment:
Could you take a look at https://github.com/InnoD-WebTier/irc/blob/master/pages/events.js and follow the format there?  Specifically, have .page.page--events wrap Helmet and a separate title div